wordpress-mobile/WordPress-iOS
# Refactoring the Create FAB into coordinator

#13320 

This PR:
- Refactors code from extensions in WPTabBarController into a coordinator object
- Adds functionality to open a new Site Page in the editor from the button (this is on by default, while the Post is commented out. The coming action sheet will allow either to be selected)

The coordinator will also manage the action sheet presentation. Those additions involve several hundred more lines of code, so I wanted to separate them so the PR stayed small enough to review.

To test:

- Go to the Blog Detail screen and tap the button
- Check that the editor posts a Page
- Check that the button hides/shows as expected on all other screens

There should be no visual difference between this change and `develop`

PR submission checklist:
